_posts/2022-02-25-roadmap-proposal.md

```diff
@@ -36,17 +36,27 @@ Taking the above into consideration, here’s how I think the majors and minors
 |Version	|Feature Freeze	|Release Date	|
 |-------------	|-------------	|-------------	|
 |1.3.0	|March 9th	|March 15th	|
-|2.0.0 RC	|Core: March 21st|April 26th	|
-| 	|Plugins and Clients: April 18th	| 	|
+|1.3.2  |April 29 2022 | May 05 2022 |
+|2.0.0 RC	|Core: March 21st|	|
+| 	|Plugins and Clients: April 18th	| ~~April 26th~~ April 28th	|
```

Comment:
Looks like the 26th / 28th update was moved from Core to the plugins. Probably needs to be moved back up to Core.

Reply:
I think it works better on the following line (since we can't release until the plugins are in).   So feature, not a bug?  :)   WDYT?

Reply:
Sounds OK to me - I'll push it through.